applications/nrf_desktop/README.rst

```diff
@@ -1421,15 +1421,20 @@ The nRF Desktop devices use :ref:`Zephyr's Bluetooth API <zephyr:bluetooth>` to
 This API is used only by the application modules that handle such connections.
 The information about peer and connection state is propagated to other application modules using :ref:`app_event_manager` events.
 
-The nRF Desktop devices come in the following types:
+The :ref:`CONFIG_DESKTOP_BT <config_desktop_app_options>` Kconfig option enables Bluetooh connectivity support in the nRF Desktop.
```

Comment:
This is already mentioned in the Bluetooth configuration section

Reply:
I think that it may still be worth to mention it here as user may not always read whole documentation

Reply:
Apart from that, it may work as nice one sentence of general introduction